spec/fixtures/constants.rb

```diff
@@ -568,6 +568,7 @@
             ~~ ^~~~~~~
 )
 
+LONG_FAILURE_ERROR = File.read(File.expand_path("../long_failure.txt", __FILE__))
```

Comment:
Line is too long. [81/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.